### PR TITLE
Add relaxed security command line flag

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
This flag, if provided to the server via command line, will allow to perform potentially insecure actions in client drivers, like direct access to adb shell in Android driver or command line access in OSX driver.

### CODE REVIEW

File: lib/appium.js [hunk 1]
```diff
@@ -28,7 +28,11 @@ class AppiumDriver extends BaseDriver {
     // the main Appium Driver has no new command timeout
     this.newCommandTimeoutMs = 0;
 
-    this.args = args;
+    this.args = Object.assign({}, args);
+    if (this.args.relaxedSecurityEnabled) {
+      this.relaxedSecurityEnabled = true;
+      delete this.args.relaxedSecurityEnabled;
```
Comment (imurchie) [inline lib/appium.js]: Why this extra step?

**Review by imurchie — COMMENTED** [review]
One question about the implementation. I would also like to see some extra logging of the fact that this flag has been turned on and its implications, at startup time.

File: lib/appium.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/appium.js]: because this option should not be in capabilities. One anyway cannot change it from there

Comment (imurchie) [reply lib/appium.js]: But only the default capabilities from the command line arguments make it into the caps sent to the driver (https://github.com/appium/appium/blob/master/lib/appium.js#L176-L177)

Comment (mykola-mokhnach) [reply lib/appium.js]: ok, makes sense

**Comment by mykola-mokhnach** [comment]
> One question about the implementation. I would also like to see some extra logging of the fact that this flag has been turned on and its implications, at startup time.

Yes, added more logging

**Review by imurchie — APPROVED** [review]

**Pull request merged**